context_legacy.go

```diff
@@ -79,6 +85,32 @@ func ctxSetToken(req *http.Request, token []byte) *http.Request {
 	return req
 }
 
+func ctxSetSent(req *http.Request) {
+	cmMutex.Lock()
+	defer cmMutex.Unlock()
+
+	ctx, ok := contextMap[req]
+	if !ok {
```

Comment:
I think it would be okay to assume context already exists here, like `Reason()` does.
https://github.com/justinas/nosurf/blob/4d86df7a4affa1fa50ab39fb09aac56c3ce9c314/context_legacy.go#L87-L90